_components/accordions.md

```diff
@@ -80,7 +79,7 @@ title: Accordions
         Clicking anywhere on the header (not just on the icon) triggers expansion or collapse of that section.
       </li>
       <li>
-        Any interactive element within each section should be placed a minimum of 2 mm from the header to avoid accidentally triggering the wrong interaction.
+        Any interactive element within each section should be placed a minimum of 10 pixels from the header to avoid accidentally triggering the wrong interaction.
```

Comment:
This one is tricky. The exact pixel recommendation varies based on device's pixel density and touch precision (which vary substantial device to device). In 5c5f14c, I made this a general principle using @carodew's suggestions. Thoughts?